include/LeMonADE/core/MoleculesRead.h

```diff
@@ -152,6 +153,79 @@ void ReadBonds<IngredientsType>::execute()
   std::cout << "ReadBonds:execute() : done " <<std::endl;
 }
 
+/***********************************************************************/
+/**
+ * @class ReadBreaks
+ * @brief Handles BFM-File-Reads \b !remove_bonds.
+ *
+ * @tparam IngredientsType Ingredients class storing all system information.
+ **/
+template < class IngredientsType >
+class ReadBreaks: public ReadToDestination < IngredientsType >
+{
+ public:
+	//! Empty Constructor, but delegates call to the Feature
+	ReadBreaks(IngredientsType& destination):ReadToDestination< IngredientsType > (destination){}
+
+	void execute();
+};
+/***********************************************************************/
+/**
+ * @brief Executes the reading routine to extract \b !remove_bonds.
+ *
+ * @throw <std::runtime_error> if broken bond could not be read.
+ **/
+
+template < class IngredientsType >
+void ReadBreaks<IngredientsType>::execute()
```

Comment:
To be more precise and comparable to ReadAddBonds/WriteRemoveBonds I really prefere to rename these classes
```suggestion
void ReadRemoveBonds<IngredientsType>::execute()
```